src/styles/blocks/card.scss

```diff
@@ -0,0 +1,119 @@
+.card {
+  border-radius: 5px;
+  border: 1px solid $border-color;
+
+  background: $white;
+  padding: 16px;
+  transition: transform 0.3s ease;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    &--position {
+      margin: 16px auto 0;
+    }
+  }
+
+  &__title {
+    color: $primary-text-color;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    transition: color 0.3s ease;
+
+    &--position {
+      margin-top: 40px;
+      margin-bottom: 0;
+    }
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    &--position {
+      margin-top: 4px;
+    }
+  }
+
+  &__review {
+    @include addSpaceBetween;
+
+    &--position {
+      margin-top: 16px;
+    }
+
+    &-count {
+      display: flex;
+      height: 16px;
+      justify-content: flex-end;
+      align-items: center;
+      padding: 0;
+
+      color: $primary-text-color;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+    }
+  }
+
+  &__price {
+    @include addSpaceBetween;
+    height: 18px;
+
+    &--position {
+      margin-top: 24px;
+    }
+
+    &-title {
+      color: $secondary-text-color;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-bucks {
+      color: $primary-text-color;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    border-radius: 5px;
+    width: 166px;
+    height: 40px;
+
+    color: $white;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-transform: uppercase;
+    text-decoration: none;
+    background: $button-color;
+    transition: border 0.3s ease, color 0.3s ease, background-color 0.3s ease;
+
+    &--position {
+      margin-top: 16px;
+    }
+
+    &:hover {
+      background: $white;
+      color: $button-color;
+      border: $button-color 1px solid;
```

Comment:
https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md
9. [STYLES] - Don't add new border to the element on hover. Add default transparent border of the same width, and change its color on :hover"